### PR TITLE
chore: enable full CI run

### PR DESCRIPTION
**Pull request by carllerche**
Re-enable commented out CI runs.

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ freebsd_instance:
 task:
   name: FreeBSD 12.0
   env:
-    LOOM_MAX_DURATION: 10
+    LOOM_MAX_PREEMPTIONS: 2
```
Comment (LucioFranco) [inline .cirrus.yml]: Are we going to set this back to 10? or did loom change so that 2 is enough to cover for most CI cases?

Comment (carllerche) [reply .cirrus.yml]: They are very different measurements :) max preemptions of 2 is much better than max duration of 10

Comment (LucioFranco) [reply .cirrus.yml]: Well good job to me I didnt even notice one was duration and one was preemptions ha!

File: tokio-tcp/tests/tcp.rs [hunk 2]
```diff
@@ -1,114 +1,77 @@
...
+        assert_ok!(theirs.peer_addr())
+    );
+    assert_eq!(
+        assert_ok!(theirs.local_addr()),
+        assert_ok!(mine.peer_addr())
+    );
 }
 
+/*
+ * TODO: bring this back once TCP exposes HUP again
```
Comment (LucioFranco) [inline tokio-tcp/tests/tcp.rs]: is there an issue connected to this?

Comment (carllerche) [reply tokio-tcp/tests/tcp.rs]: #1209 includes a note for it.

**Pull request merged**